query/src/main/scala/filodb/query/exec/rangefn/AggrOverTimeFunctions.scala

```diff
@@ -671,16 +676,17 @@ class QuantileOverTimeChunkedFunctionD(funcParams: Seq[Any]) extends QuantileOve
   }
 }
 
-class QuantileOverTimeChunkedFunctionL(funcParams: Seq[Any])
+class QuantileOverTimeChunkedFunctionL(funcParams: Seq[FuncArgs])
   extends QuantileOverTimeChunkedFunction(funcParams) with ChunkedLongRangeFunction {
   require(funcParams.size == 1, "quantile_over_time function needs a single quantile argument")
+  require(funcParams.head.isInstanceOf[StaticFuncArgs], "quantile parameter must be a number")
   require(funcParams.head.isInstanceOf[Number], "quantile parameter must be a number")
```

Comment:
Wait, shouldn't this line be removed?  funcParams.head can't be both `StaticFuncArgs` and a `Number`

Reply:
Ohh yes. I missed this. Thanks for letting me know! We don't use QuantileOverTimeChunkedFunctionL so the test cases were not failing